doc/Programming-guide/Key-platform-features/Configuration-management/index.md

```diff
@@ -761,7 +763,7 @@ class UserConfigurationReceiver : public IConfigurationReceiver {
 public:
     void printConfiguration(const KaaRootConfiguration &configuration)
     {
-        std::cout << "Current configuration: " << configuration << std::endl;
+        // Add the code which prints the configuration schema.
```

Comment:
Why do we need this method for simple example?
The configuration can be printed in `onConfigurationUpdated()` for sake of simplicity.